sync/class.jetpack-sync-module-themes.php

```diff
@@ -354,9 +376,10 @@ function sync_sidebar_widgets_actions( $old_value, $new_value ) {
 			 *
 			 * @since 4.9.0
 			 *
-			 * @param array $sidebar, Sidebar id got changed
+			 * @param array $moved_to_inactive, Array of widgets id that moved to inactive id got changed
+			 * @param array $moved_to_inactive, Array of widgets names that moved to inactive id got changed Since 5.0.0
 			 */
-			do_action( 'jetpack_widget_moved_to_inactive', $moved_to_inactive );
+			do_action( 'jetpack_widget_moved_to_inactive', $moved_to_inactive, array_map( array( $this, 'get_widget_name' ), $moved_to_inactive ) );
```

Comment:
Consider renaming $moved_to_inactive to $moved_to_inactive_ids
Consider reducing `array_map( array( $this, 'get_widget_name' ), $moved_to_inactive ) )` to a variable called $moved_to_inactive_names and passing it as an argument to do_action